### ISSUE TITLE
Failing to get a wildcard certificate

### ISSUE THREAD

**Issue by hpinhal** [adferrand/dnsrobocert#90]
Hi, 

I'm trying to reconfigure our setup with the v3 instructions but I'm failing to acquire a wildcard certificate.

The logs are not very detailed about what's going on, and this is the only non-info output I'm getting.

```
certificates_1  | 2020-03-28 14:44:40 dc882ea251f7 dnsrobocert.core.main[1] ERROR An error occurred while processing certificate config `{'profile': 'digitalocean', 'domains': ['my-domain.dev', '*.my-domain.dev']}`:
certificates_1  | Following command with non-zero errorcode (1):
certificates_1  | /usr/bin/python3 -m dnsrobocert.core.certbot certonly -n --config-dir /etc/letsencrypt --work-dir /etc/letsencrypt/workdir --logs-dir /etc/letsencrypt/logs --manual --preferred-challenges=dns --manual-auth-hook "/usr/bin/python3 -m dnsrobocert.core.hooks -t auth -c \"/tmp/tmprewex7ls/dnsrobocert-runtime.yml\" -l \"my-domain.dev\"" --manual-cleanup-hook "/usr/bin/python3 -m dnsrobocert.core.hooks -t cleanup -c \"/tmp/tmprewex7ls/dnsrobocert-runtime.yml\" -l \"my-domain.dev\"" --manual-public-ip-logging-ok --expand --deploy-hook "/usr/bin/python3 -m dnsrobocert.core.hooks -t deploy -c \"/tmp/tmprewex7ls/dnsrobocert-runtime.yml\" -l \"my-domain.dev\"" --server https://acme-staging-v02.api.letsencrypt.org/directory --cert-name my-domain.dev -d my-domain.dev -d *.my-domain.dev
```

Is there something to gather from this?

**Comment by adferrand** [comment]
Hello! From these logs certbot clearly failed acquiring the certificate, but this could be for a lot of different reasons.

I am surprised that you do not have any output from what happened in certbot, since normally it is displayed by dnsrobocert. Could you post the entire log

If you want to avoid any sensitive data leak displayed in a public place, I created a Spectrum community for user and dev support. You will be able to PM me there: https://spectrum.chat/dnsrobocert

**Comment by hpinhal** [comment]
I did a little more digging and it turned out to be a failed authorisation. I am using a digital ocean provider. I really cannot say why, but this morning it simply worked.

I assume this may have been a combo of misconfiguration and rate limiting on LE.

On the logging part, it's rather erratic. Sometimes I get the certbot output for a created certificate, sometimes don't. This morning is a perfect example of that. I ran the script, which has 5 certificates configured. I know four of them were still valid but I would expect to see in the terminal the "no renewal needed" message, which I didn't get. The fifth was the wildcard that wouldn't work yesterday. It got the certificate but no log output.

I think the original problem was myself. Therefore I am closing this issue.

If you'd like to follow up on the logging _problems_ let's do that in a separate issue.

Cheers!
